### PR TITLE
Feature/join tables

### PR DESCRIPTION
**Pull request by majdalkilany**
Modify animals table 
Create new 2 tables 
Insert the data related to each one of them 

queries screenshots  

first part of the queries

second part of the queries

### CODE REVIEW

File: schema.sql
Comment (aamirkhan2478) [inline schema.sql:1]: - [ ] Please add one ew table with the name owners with the following columns:

```sql
CREATE TABLE owners (
   id int NOT NULL IDENTITY(1,1),
   full_name varchar(255),
   age int,
   PRIMARY KEY (id)
   );
```

- [ ] Please add new table with the name `species` with the following columns:

```sql
   CREATE TABLE species (
   id int NOT NULL IDENTITY(1,1),
   name varchar(255),
   PRIMARY KEY (id)
   );
```

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @majdalkilany, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by majdalkilany** [comment]
The Tables were added  to the schema

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+   PRIMARY KEY (id)
+   );
+
+
+      CREATE TABLE species (
+   id int NOT NULL IDENTITY(1,1),
+   name varchar(255),
+   PRIMARY KEY (id)
+   );
+
```
Comment (Qoosim) [inline schema.sql]: - [ ] Great job implementing some required changes. However, there's one table yet to be added. (i.e) `owners` table. Kindly add it and remove the duplicated `species` table.
```
CREATE TABLE owners (
   id int NOT NULL IDENTITY(1,1),
   full_name varchar(255),
   age int,
   PRIMARY KEY (id)
 );
```

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @majdalkilany, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  Implemented some required changes. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by majdalkilany** [comment]
Remove the duplicate table

**Review by topeogunleye — APPROVED** [review]
### Hi @majdalkilany, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Added owners table. ✔️ 
- Removed duplicate species table. ✔️ 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — COMMENTED** [review]
Hi @majdalkilany, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

## Highlights
- Great job, your project is error free :+1: 
- Nice work you meet all the requirements :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**